<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="RenderArea" name="renderArea" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>164</width>
      <height>215</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAstroid">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>86</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Astroid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCycloid">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>44</y>
      <width>87</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cycloid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHuygens">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>78</y>
      <width>144</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Huygens Cycloid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHypo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>112</y>
      <width>123</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Hypo Cycloid</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>217</y>
      <width>157</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Interval Length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinIntervalLength">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>184</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinScale">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnLine">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>146</y>
      <width>68</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>124</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Step count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinCount">
       <property name="maximum">
        <number>512</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header>renderarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
